aiida/sphinxext/tests/test_workchain.py

```diff
@@ -15,12 +15,16 @@
 from os.path import join, dirname
 
 import six
-import pytest
+import pytest  # pylint: disable=unused-import
```

Comment:
If it is unused, I guess simply remove it?

Reply:
hm, I wasn't sure whether it is unused in the sense that there's all this magic needed to inject the fixtures.
I'll try removing it.